README.md

```diff
@@ -81,3 +81,5 @@ If the filename ends with `.url` suffix, the content will be processed as a URL
 | `WATCH_SERVER_TIMEOUT`     | polite request to the server, asking it to cleanly close watch connections after this amount of seconds ([#85](https://github.com/kiwigrid/k8s-sidecar/issues/85))                                                                                                                                                                  | false    | `60`                                      | integer |
 | `WATCH_CLIENT_TIMEOUT`     | If you have a network outage dropping all packets with no RST/FIN, this is how many seconds your client waits on watches before realizing & dropping the connection. You can keep this number low. ([#85](https://github.com/kiwigrid/k8s-sidecar/issues/85))                                                                       | false    | `66`                                      | integer |
 | `IGNORE_ALREADY_PROCESSED` | Ignore already processed resource version. Avoid numerous checks on same unchanged resource. req kubernetes api >= v1.19                                                                                                                                                                                                            | false    | `false`                                   | boolean |
+| `LOG_LEVEL`                | Set the logging level. (DEBUG, INFO, WARN, ERROR, CRITICAL)                                                                                                                                                                                                                                                                         | false    | `INFO`                                    | string  |
+| `LOG_FORMAT`               | Set a log format. (JSON or LOGFMT)                                                                                                                                                                                                                                                                                                  | false    | `LOGFMT`                                  | string  |
```

Comment:
How about making this config then " ... `JSON` or a format string for the [default formatter](https://docs.python.org/3/library/logging.html#logging.Formatter)" ?

Reply:
You mean make Json the default format ? 

Absolutely, I personally like LogFmt because it's more human readable and les verbose that Json. But Json is probably more versatile for parsing by log aggregator (Loki, ELK ...)

Reply:
I changed to have `JSON` has the default